server/src/main/java/org/spine3/server/aggregate/AggregateRepository.java

```diff
@@ -243,6 +231,55 @@ public void dispatch(Command request) throws IllegalStateException {
         commandStatusService.setOk(commandId);
     }
 
+    @SuppressWarnings("ChainOfInstanceofChecks")        // it's a rare case of handing an exception, so we are OK.
```

Comment:
Previously the suppression looked as `//noinspection ...`. It was declared closer to the `instanceof` chain. As we are aiming not only for IDEA codestyle tweaks (`//noinspection` works fine), but also for tools like PMD, we need to use a standard-based suppression mechanism, which is `@SuppressWarnings(..)`.

However, it was not possible just to swap the previously declared `//noinspection` with a descent `@SuppressWarnings(...)` at the same spot due to Java syntax rules. 

That's why I moved the `@SuppressWarnings` to the next best place, which is the method annotation.